definitions/variable/tag_electricity_input_types.yaml

```diff
@@ -284,7 +284,10 @@
   - Hydro|Pumped Storage:
       description: hydropower systems with short-term storage capacity
         (including pumping turbines)
-      notes: This hydropower system has two water reservoirs at different elevations.
+      notes: This variable should only include electricity generation from natural
```

Comment:
I don't understand this..... We cannot 'separate' electricity generated from the natural inflows and electricity generated from water which has previously been pumped........... A pumped hydro unit will generate electricity (xMW at each hour), and pump water while consuming electricity (yMW at each hour); Note that it is possible to 'pump' and 'turbine' simultaneously (and this has proven very efficient, in particular regarding system services, so numerous pumped storage units are currently being given this possibility); Also I do not understand why restrictions are detailed here about a 'variable' while this is only a 'tag' list.....

Reply:
to complement the above, a Pumped Hydro system could also have natural inflows, thus could both generate additionnal electricity AND 'discharge ' stored electricity -(ir generate electricity from water which was previously pumped); My model allows that (but it is not currently used as we do not have data for inflows to pumped hydro storages.....)

Reply:
I removed my attempt at disambiguation.